www/include/eventLogs/viewLog.php

```diff
@@ -424,7 +434,8 @@ function log_4_engine_page(id, formu, num) {
     }
     var _host = <?php echo !empty($user_params["log_filter_host"]) ? $user_params["log_filter_host"] : 'false'; ?>;
     var _service = <?php echo !empty($user_params["log_filter_svc"]) ? $user_params["log_filter_svc"] : 'false'; ?>;
-    var _engine = <?php echo $engine; ?>;
+    // Casting engine variable so that it can be interpreted corretly in JS
+    var _engine = <?php echo (int)$engine; ?>;
```

Comment:
you can use "<= ... >" for echo

Reply:
Ok thanks :100: I'll know it for next time :+1: